src/main/resources/swagger.api/schemas/instanceIterationRequest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Start new instance iteration job",
+  "type": "object",
+  "properties":
+  {
+    "eventType":
+    {
+      "description": "Type of events to be published",
+      "type": "string"
+    },
+    "topicName":
+    {
+      "description": "Name of Kafka topic to publish events to",
+      "type": "string"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
Not sure if this DTO definition should be part of API schema, it doesn't seem to be used for any request/response